wiki/Performance_Troubleshooting/en.md

```diff
@@ -0,0 +1,71 @@
+
+### Low frame rate
+During gameplay, the frame rate is unable to keep up, resulting in jittery and awkward movement of objects on the screen. This can also result in higher perceived input latency. When diagnosing or reporting, it may help to:
+
+- Check whether it still happens straight after a PC restart.
+- Try closing any application which shows overlays on the game (Discord, Steam, etc.).
+- Make sure you're not forcing vsync on at your display drivers (it should be set to "off" or "application preference").
+- Try changing frame limiter and see if it improves the situation.
+- Make sure you are running osu! in exclusive (not borderless) fullscreen mode. This will always improve performance.
+
+## Frame Limiters
+osu! offers multiple choices of frame limiters. Each of these have pros and cons. Note that regardless of setting, osu! will limit frame rate at menus to conserve on CPU usage.
+
+### VSync
+This frame limiter relies on a driver provided vertical synchronising mechanism. It guarantees no screen tearing, but adds 1-2 frames worth of latency due to the requirement of buffering frames before display.
+
+Generally not recommended for 60hz displays. It can become more useful on 120hz+, but with GSync and FreeSync monitors, the requirement to use VSync to avoid tearing is lifted.
+
+### Power Saving
+This frame limiter aims to save on power usage while providing a consistent game experience. Will run at 2x monitor refresh rate and attempt to keep a consistent frame pacing.
+
+Recommended for older PCs or for conserving battery on a laptop.
+
+### Optimal
+This frame limiter attempts to keep a frame rate 8x your refresh rate, with a maximum of 960hz. This is our recommended frame limiter for users who don’t want to run their CPU/GPU to extremities, while maintaining high levels of performance resulting in low input latency.
+
+Can be preferred to unlimited in order to reduce the number of frames rendered, in turn leading to less potential for garbage collection overhead (which can result in spike frames).
+
+### Unlimited
+This is a frame limiter which does nothing to limit the frame rate. osu! will render as fast as your CPU/GPU can.
+
+While this may seem like the best option (and indeed can be, if your hardware can support it), there are a few things you should consider:
+
+- By rendering more frames, there’s a higher potential of overhead from garbage collection
+- By pushing your CPU and GPU to the limit, there’s a chance they will throttle, lowering your overall performance.
+- GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
+
+Even so, if heat generation and working your hardware is not considered an issue, this may be the recommended option over optimal for users which get more stable frame times by using it.
```

Comment:
```suggestion
Even so, if heat generation and working your hardware are not considered an issue, this may be the recommended option over optimal for users which get more stable frame times by using it.
```

Reply:
this whole sentence actually reads quite convoluted. needs to be refactored.

Reply:
kinda agree